pkg/conditions/condition_types.go

```diff
@@ -37,29 +37,42 @@ const (
 	PoolManagerRunningReason ConditionReason = "PoolManagerRunning"
 	PoolManagerFailureReason ConditionReason = "PoolManagerFailure"
 
-	SecretReference                ConditionType   = "SecretReference"
-	FetchingSecretRefSuccessReason ConditionReason = "FetchingSecretRefSuccess"
-	FetchingSecretRefFailedReason  ConditionReason = "FetchingSecretRefFailed"
+	WebhookSecretReference                ConditionType   = "WebhookSecretReference"
+	FetchingWebhookSecretRefSuccessReason ConditionReason = "FetchingWebhookSecretRefSuccess"
+	FetchingWebhookSecretRefFailedReason  ConditionReason = "FetchingWebhookSecretRefFailed"
 
-	CredentialsReference                ConditionType   = "CredentialsReference"
-	FetchingCredentialsRefSuccessReason ConditionReason = "CredentialsRefSuccess"
-	FetchingCredentialsRefFailedReason  ConditionReason = "CredentialsRefFailed"
+	GithubCredentialsReference                ConditionType   = "GithubCredentialsReference"  // #nosec G101
+	FetchingGithubCredentialsRefSuccessReason ConditionReason = "GithubCredentialsRefSuccess" // #nosec G101
+	FetchingGithubCredentialsRefFailedReason  ConditionReason = "GithubCredentialsRefFailed"  // #nosec G101
 )
 
 // Credential Conditions
 const (
-	EndpointReference                ConditionType   = "EndpointReference"
-	FetchingEndpointRefSuccessReason ConditionReason = "FetchingEndpointRefSuccess"
-	FetchingEndpointRefFailedReason  ConditionReason = "FetchingEndpointRefFailed"
+	GithubEndpointReference                ConditionType   = "GithubEndpointReference"
+	FetchingGithubEndpointRefSuccessReason ConditionReason = "FetchingGithubEndpointRefSuccess"
+	FetchingGithubEndpointRefFailedReason  ConditionReason = "FetchingGithubEndpointRefFailed"
 )
 
+// Runner Conditions
 const (
-	GarmServerNotReconciledYetMsg  string = "GARM server not reconciled yet"
-	CredentialsNotReconciledYetMsg string = "credentials not reconciled yet"
-	DeletingEnterpriseMsg          string = "Deleting enterprise"
-	DeletingOrgMsg                 string = "Deleting organization"
-	DeletingRepoMsg                string = "Deleting repository"
-	DeletingPoolMsg                string = "Deleting pool"
-	DeletingEndpointMsg            string = "Deleting endpoint"
-	DeletingCredentialsMsg         string = "Deleting credentials" // #nosec G101
+	RunnerReadyReason    ConditionReason = "RunnerIdleAndRunning"
+	RunnerNotReadyReason ConditionReason = "RunnerNotReadyYet"
+	RunnerErrorReason    ConditionReason = "RunnerProvisioningError"
+)
+
+const (
+	GarmServerNotReconciledYetMsg     string = "GARM server not reconciled yet"
+	CredentialsNotReconciledYetMsg    string = "Github Credentials Ref not reconciled yet" // #nosec G101
+	GithubEndpointNotReconciledYetMsg string = "Github Endpoint Ref not reconciled yet"    // #nosec G101
+	WebhookSecretNotReconciledYetMsg  string = "Webhook Secret Ref not reconciled yet"     // #nosec G101
```

Comment:
i would propose to either not use abbreviations
```suggestion
	CredentialsNotReconciledYetMsg    string = "Github credentials reference not reconciled yet" // #nosec G101
	GithubEndpointNotReconciledYetMsg string = "Github endpoint reference not reconciled yet"    // #nosec G101
	WebhookSecretNotReconciledYetMsg  string = "Webhook secret reference not reconciled yet"     // #nosec G101
```

or name it like the name from the spec:

```suggestion
	CredentialsNotReconciledYetMsg    string = "GithubCredentialsRef not reconciled yet" // #nosec G101
	GithubEndpointNotReconciledYetMsg string = "GithubEndpointRef not reconciled yet"    // #nosec G101
	WebhookSecretNotReconciledYetMsg  string = "WebhookSecretRef not reconciled yet"     // #nosec G101
```